samples/subsys/usb/testusb/README.rst

```diff
@@ -0,0 +1,58 @@
+.. _testusb-app:
+
+Testusb application sample
+##########################
+
+The testusb implements a loopback function. This function can be used
+to test USB device drivers and the device stack connected to Linux Host
+and has a similar interface to "Gadget Zero" of the Linux kernel.
+The userspace tool testusb [3] is needed to start the tests.
+
+Building and flashing
+*********************
+
+Follow the general procedure for building and flashing Zephyr device.
+
+Testing
+*******
+
+To run USB Chaper 9 tests:
+
+#. Load the usbtest module on the Linux Host
```

Comment:
Sounds like ``usbtest`` and ``testusb`` two different things.  It would be good to explain the difference (otherwise it sounds like a typo).

Reply:
usbtest is Linux kernel module, testusb is a command line tool